server/modules/auth/auth.middleware.ts

```
@@ -44,7 +43,7 @@ export const protect = async (
 
   try {
     // Verify token
-    const decoded: any = jwt.verify(token, process.env.JWT_SECRET!);
+    const decoded = jwt.verify(token, process.env.JWT_SECRET!) as JwtPayload;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)